src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+public class Money {
+
+    public static final long LOTTO_PRICE = 1000L;
+    private final long money;
+
+    public Money(long money) {
+        validateMoney(money);
+        this.money = money;
+    }
+
+    private void validateMoney(long money) {
+        if (money < 0) {
+            throw new IllegalArgumentException("돈이 음수일 수 없습니다.");
```

Comment:
예외 메시지를 작성할 땐 어떤 데이터가 입력되었는지 작성하는 습관을 기르면 나중에 매우 큰 보답을 받을 수 있습니다
```suggestion
            throw new IllegalArgumentException(String.format("돈이 음수일 수 없습니다. 입력된 금액 : %d", money));
```